tools/tools.mk

```diff
@@ -4,7 +4,7 @@ $(TOOLS_BINDIR)/makefat: $(TOOLS_DIR)/go.mod
 	cd $(TOOLS_DIR) && GOBIN=$(TOOLS_BINDIR) go install github.com/randall77/makefat
 
 $(TOOLS_BINDIR)/golangci-lint: $(TOOLS_DIR)/go.mod
-	cd $(TOOLS_DIR) && GOBIN=$(TOOLS_BINDIR) go install github.com/golangci/golangci-lint/cmd/golangci-lint
+	cd $(TOOLS_DIR) && GOBIN=$(TOOLS_BINDIR) go install github.com/golangci/golangci-lint/v2/cmd/golangci-lint@v2.1.6
```

Comment:
`cd $(TOOLS_DIR) && GOBIN=$(TOOLS_BINDIR) go install github.com/golangci/golangci-lint/v2/cmd/golangci-lint` is enough, this will let `go` use the version specified in `tools/go.mod`